pkg/generator/generator.go

```diff
@@ -112,26 +165,7 @@ func (g *XGenerator) GenerateXRD() (*c.CompositeResourceDefinition, error) {
 				Plural:     "composite" + plural,
 				Categories: g.generateCategories(),
 			},
-			Versions: []c.CompositeResourceDefinitionVersion{
-				{
-					Name:          g.Version,
-					Referenceable: version.Storage,
```

Comment:
since we have multiple versions in provider crds we cannot use `version.Storage` and `version.Served` because we generating only one version in xrd so we need to set here true / true

```
diff --git a/pkg/generator/generator.go b/pkg/generator/generator.go
index 63e2281..2ef9218 100644
--- a/pkg/generator/generator.go
+++ b/pkg/generator/generator.go
@@ -115,8 +115,8 @@ func (g *XGenerator) GenerateXRD() (*c.CompositeResourceDefinition, error) {
                        Versions: []c.CompositeResourceDefinitionVersion{
                                {
                                        Name:          g.Version,
-                                       Referenceable: version.Storage,
-                                       Served:        version.Served,
+                                       Referenceable: true,
+                                       Served:        true,
                                        Schema: &c.CompositeResourceValidation{
                                                OpenAPIV3Schema: runtime.RawExtension{
                                                        Object: &unstructured.Unstructured{
```

we don't need an configuration option for generate.yaml because we have only one version